### PR TITLE
Asher hbv

### PR DESCRIPTION
**Pull request by asherps**
Created an ohio-specific acute hbv estimate for 2021.

### CODE REVIEW

File: pathogen_properties.py [hunk 1]
```diff
@@ -207,7 +211,7 @@ def target_location(
         countries = set(i.country for i in inputs if i.country)
         states = set(i.state for i in inputs if i.state)
         counties = set(i.county for i in inputs if i.county)
-
+        print(states)
```
Comment (jeffkaufman) [inline pathogen_properties.py:214]: rm

File: pathogen_properties.py [hunk 2]
```diff
@@ -255,6 +270,9 @@ class Population(Taggable):
 
     people: float
 
+    def __sub__(self, other: "Population") -> "Population":
+        return dataclasses.replace(self, people=self.people - other.people)
```
Comment (jeffkaufman) [inline pathogen_properties.py:274]: `other` needs to end up in `inputs`

File: pathogens/rhinovirus.py [hunk 3]
```diff
@@ -0,0 +1,264 @@
+from pathogen_properties import *
```
Comment (jeffkaufman) [inline pathogens/rhinovirus.py:1]: Your changes to rhinovirus.py don't belong in this PR.

In general, when you start a new PR that should look like:

```
git checkout main
git branch -b asher-some-feature
...
```

I suspect you skipped the `git checkout main` here?

Comment (jeffkaufman) [reply pathogens/rhinovirus.py:1]: Same with your `pathogen_properties.py` changes.  I think this PR should only include `hbv.py`?

**Review by jeffkaufman — CHANGES_REQUESTED** [review]

File: pathogens/hav.py
Comment (jeffkaufman) [inline pathogens/hav.py:1]: Why are you deleting `hav.py`?  This PR shouldn't affect `hav.py` either way.

File: pathogens/hbv.py [hunk 4]
```diff
@@ -66,9 +66,25 @@
 )
 
 
+ohio_acute_incidence_2021 = IncidenceRate(
+    # This source reports both an acute and a chronic incidence. I think it
+    # makes sense to add these 2 numbers for an acute estimate, since chronic
+    # cases still have an acute phase in which they shed.
```
Comment (jeffkaufman) [inline pathogens/hbv.py:72]: @simonleandergrimm does this seem reasonable to you?  (Looks fine to me)

**Review by jeffkaufman — CHANGES_REQUESTED** [review]

File: pathogens/hbv.py (hunk 4 above)
Comment (asherps) [reply pathogens/hbv.py:72]: I think if this seems reasonable to you, I think it's good to go? I don't think there are any other changes I need to make (there was no reason to remove hav, as jeff noted above, but it shouldnt matter either way if this is merged into main)

Comment (simonleandergrimm) [reply pathogens/hbv.py:72]: This isn't needed. If you look at the table more specifically, the table gives **acute** and **total** numbers. The authors then say that
```
Total hepatitis equals all hepatitis B cases,both "acute" and "chronic".
```
So just using total is the way to go (which Asher is already doing). I told @asherps to change the wording of the comment to not make it sound like there is manual addition.

File: pathogens/hbv.py [hunk 5]
```diff
@@ -66,9 +66,30 @@
...
 
+ohio_total_incidence_2021 = IncidenceRate(
+    # This source reports both an acute and a total incidence. The total
+    # incidence is the sum of the acute and chronic incidences. I think it
+    # makes sense to report the total incidence, since chronic
+    # cases still have an acute phase in which they shed. The number of
+    # chronic cases has some lag attached, since chronic cases are probably
+    # not identified during their acute phase. However, the reported chronic
+    # incidence in any given year should still be around the true number of
+    # new chronic cases for that year.
```
Comment (simonleandergrimm) [inline pathogens/hbv.py:77]: What do you mean by true number of new chronic cases here?

Comment (simonleandergrimm) [reply pathogens/hbv.py:77]: I would also shorten this paragraph to one or two sentences, it's a bit wordy atm.

**Review by simonleandergrimm — CHANGES_REQUESTED** [review]

File: pathogens/hbv.py [hunk 6]
```diff
@@ -66,9 +66,29 @@
...
 
 
+ohio_total_incidence_2021 = IncidenceRate(
+    # This source reports both an acute and a total (acute+chronic) incidence.
+    # I think it makes sense to report the total incidence, since chronic
+    # cases still have an acute phase in which they shed. The number of
+    # chronic cases in any given year has some lag attached, since chronic
+    # cases are probably not identified during their acute phase. However, the
+    # reported chronic incidence in any given year should still be
+    # approximately the number of new chronic cases for that year.
```
Comment (simonleandergrimm) [inline pathogens/hbv.py:76]: We talked about this yesterday, but I am unsure what you are arguing for or against here, could you explain?

```
However, the reported chronic incidence in any given year should still be approximately the number of new chronic cases for that year.
```
What do you mean by this? I would think reported incidence very likely underestimates the true number of true cases.

**Review by simonleandergrimm — COMMENTED** [review]
The estimate looks good, I just have a question on your comment on line 70.

File: pathogens/hbv.py (hunk 6 above)
Comment (asherps) [reply pathogens/hbv.py:76]: I'm arguing that chronic cases should be counted because they're our best guess at true chronic incidence for a current year, even some of the new chronic cases identified are from years ago. I updated the comment again, see what you think

File: pathogens/hbv.py [hunk 7]
```diff
@@ -66,9 +66,29 @@
...
 
 
+ohio_total_incidence_2021 = IncidenceRate(
+    # This source reports both an acute and a total (acute+chronic) incidence.
+    # I think it makes sense to report the total incidence, since chronic
+    # cases still have an acute phase in which they shed. The number of
+    # chronic cases in any given year has some lag attached, since chronic
+    # cases are probably not identified during their acute phase, but this
+    # number still roughly represents our most recent guess at new chronic
+    # cases for 2021
```
Comment (simonleandergrimm) [inline pathogens/hbv.py:76]: Added it to my agenda to talk through this tomorrow

**Comment by simonleandergrimm** [comment]
Superseded by #122 . I will thus close this PR.
